### DISCUSSION TITLE
Ошибка с календарем в мобильной версии

### DISCUSSION THREAD

**Discussion by a1ext0** [totumonline/totum-ru-issues-and-discussions#325] [ОШБК!] [answered]
Доброго времени суток!

Обнаружил ошибку с календарём в мобильной версии. При добавлении новой строки в таблицу с мобильной версии в поле "дата" не выскакивает инпут с календариком.
![IMAGE 2023-02-02 16:43:16](https://user-images.githubusercontent.com/54662036/216309614-c742e600-2e28-4980-b669-786fcda88210.jpg)

При этом, если поле стоит обязательное, то при вводе даты в любом из форматов выскакивает ошибка, якобы поле не заполнено
![IMAGE 2023-02-02 16:42:15](https://user-images.githubusercontent.com/54662036/216309465-b1213932-c955-495a-9c75-983166315893.jpg)
![IMAGE 2023-02-02 16:42:21](https://user-images.githubusercontent.com/54662036/216309523-5edc9b52-cf4f-47c9-8e81-a671d4d65d4b.jpg)

При изменении существующей строчки всё ок работает при этом. Было опробовано на браузерах хром,файрфокс и опера - везде идентичная история. Возможно дело в настройках поля, но они почти дефолтные. Прикладываю скрин с настройками ниже
<img width="500" alt="image" src="https://user-images.githubusercontent.com/54662036/216312800-649272bf-13e7-42a5-ab5a-7b2a697e9770.png">
<img width="500" alt="image" src="https://user-images.githubusercontent.com/54662036/216312902-5e5b62f5-d8fa-4899-a268-9dc3c1ce3197.png">

Какие есть варианты решения этого вопроса?

Вопрос 2 вытекающий из этого: у одного из пользователей на ноутбуке экран с небольшим разрешением и его переводит на мобильную версию. Из-за этого он не может нормально работать с датами при добавлении. Если в мобильную версию работу с датами при добавлении не планируется включать, то есть ли способ принудительно перевести его на десктоп версию (видел у админа кнопку такую при скрытии UI админского, а у пользователей такой функции нет). Возможно подскажете сторонние варианты типа расширений, буду крайне благодарен

Заранее спасибо за ответ

**Comment by ttm-alexeypolunin** [comment]
Это какая-то старая ошибка. Мы ее чинили — обновите

![photo_2023-02-02 14 46 30](https://user-images.githubusercontent.com/55755565/216316468-7e156630-55d3-474e-9078-52ace1b0e7c8.jpeg)

Подумаем над переключением в десктоп для пользователей.

**Reply by a1ext0** [reply]
<img width="210" alt="image" src="https://user-images.githubusercontent.com/54662036/216317240-c7875187-d4ca-40ec-bbcb-6f65e595ee0e.png">
У меня вроде новая версия тотума, или есть обновление?

**Reply by ttm-alexeypolunin** [reply]
Это игра «Угадаю ошибку с 3-х пикселей».

Да, я смог повторить. Связано с предварительным открытием iframe — поправим.

**Reply by a1ext0** [reply]
Была такая догадка как раз, сори что написал, забыл про неё.
Разработчики тотум настолько круты что читают мысли 😁

**Reply by ttm-alexeypolunin** [reply]
Приходится :)

**Reply by ttm-alexeypolunin** [reply]
Скорее всего завтра будет апдейт, не уверен, что сег успеем

**Reply by ttm-alexeypolunin** [reply]
**поправили**

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Ошибка — исправлено в 4.9.52.1

**Reply by ttm-alexeypolunin** [reply]
В **4.9.52.2** изменен механизм определения мобильного интерфейса (по наличию высокоточного пойнтера называемого «мышка»). А также добавлена переключатель между интерфейсами для всех пользователей.

<img width="311" alt="Снимок экрана 2023-02-10 в 16 57 30" src="https://user-images.githubusercontent.com/55755565/218109743-b1b15e56-ef7e-4f96-b03d-3a548fb0784c.png">

Ограничение остается для экранов менее 800 px шириной — для них принудительно мобильная версия без возможности переключения. Возможно потом еще уменьшим эту цифру тк пока непонятно применение на реальных устройствах.

**Reply by a1ext0** [reply]
Добрый день! Спасибо большое, очень круто! На гите обновление когда ждать? Последний в ветке PRO вижу 4.9.52.1

**Reply by ttm-alexeypolunin** [reply]
Для PRO в понедельник
